### PR TITLE
Bumped version to 0.4.0

### PR DESCRIPTION
**Pull request by drorata**
See: https://github.com/TomAugspurger/engarde/commit/022cfa3a8aab7a227dc10c24647d5f68ade15184

### CODE REVIEW

**Comment by drorata** [comment]
@TomAugspurger correct me if I'm wrong, there's some problem with the tests, right?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% set name = "engarde" %}
-{% set version = "0.3.2" %}
+{% set version = "0.4.0" %}
 {% set sha256 = "70df13ed1470d099b359c514f8da6835ce5645c8f79b3edd537f641a8699e1db" %}
```
Comment (TomAugspurger) [inline recipe/meta.yaml]: You'll need to update the sha. I think it should be 05cb8e5299d64ac1cd277fe69815b2a611af1607d2686461b46a357210b040f2

which you can get from https://pypi.org/project/engarde/0.4.0/#files or by running `openssl dgst -sha256` on the `.tar.gz`.

Comment (drorata) [reply recipe/meta.yaml]: Done.

**Comment by TomAugspurger** [comment]
Thanks, looks good. Let me know if you notice that the workers all pass before I do.

**Comment by drorata** [comment]
@TomAugspurger All tests passed.

**Comment by TomAugspurger** [comment]
Thanks @drorata. These should trickle into the conda-forge channel throughout the day.

**Pull request merged**